mavros/include/mavros/utils.h

```diff
@@ -65,6 +64,9 @@ std::string to_string(mavlink::common::MAV_AUTOPILOT e);
 std::string to_string(mavlink::common::MAV_TYPE e);
 std::string to_string(mavlink::common::MAV_STATE e);
 std::string to_string(timesync_mode e);
+std::string to_string(mavlink::common::ADSB_ALTITUDE_TYPE e);
+std::string to_string(mavlink::common::ADSB_EMITTER_TYPE e);
+std::string to_string(const uint16_t &bitmask);
```

Comment:
Because it is generic type. How you suppose to deal when another uin16_t bit field will be needed?